wire-ios-data-model/Source/Model/Conversation/ZMConversation+Creation.swift

```diff
@@ -171,51 +171,4 @@ extension ZMConversation {
         return conversation
     }
 
-    static func fetchOneToOneTeamConversation(
```

Comment:
We don't need this anymore because we when we create a team one on one, it will be accessible via `user.oneOnOneConversation`.

Reply:
Yes you're right. I suppose could query `user.oneOnOneConversation` and if it is nil and the user is a team member, than we can fetch it, assign it, and return it. What do you think?

Reply:
> But don't we need to fetch it in order to to assign user.oneOnOneConversation? The conversation is not necessarily created by the self user, it could be created by the other user.

The code being removed here is only about fetching the conversation in the local db. Wouldn't your comment also apply to the "previous" way of handling fake "1:1" ? 

Where/when did we ever fetch or create the fake 1:1s before?

Reply:
> Yes you're right. I suppose could query `user.oneOnOneConversation` and if it is nil and the user is a team member, than we can fetch it, assign it, and return it. What do you think?

Yes, something like that. If the `oneOnOneConversation` is nil and the user is a team member then resolve the one-on-one. If the protocol resolves to proteus for the team user then fetch & assign the conversation to `user.oneOnOneConversation` or create it if doesn't exist.

Reply:
> Where/when did we ever fetch or create the fake 1:1s before?

@David-Henner the fake 1-1s are just group conversations which I guess we get when we do a slow sync or when processing update events. These conversations are in the db, and so that's we had this special fetch for 1-1s using predicates.

Reply:
> If the oneOnOneConversation is nil and the user is a team member then resolve the one-on-one. 

@typfel hmm so we could actually avoid a heavyweight migration by resolving the one on one if `oneOnOneConversation` is `nil`, either fetching a local instance or creating a new one. Right?

Reply:
> But don't we need to fetch it in order to to assign user.oneOnOneConversation? The conversation is not necessarily created by the self user, it could be created by the other user.

I don't see where we on a fresh login assign the user.oneOnOneConversation for all team users we have 1-1 conversation with.

@typfel actually I found a way we can link the user and fake 1-1 conversation together: when we update or create the group conversation (that is actually a fake 1-1) then we can check if it's a fake 1-1 by querying `conversation.conversationType == .oneOnOne` and assign the other user.

Doing it here means we will link the conversation and user for both fake 1-1s found via slow sync as well as when another user creates the 1-1 with you.